samples/sensor/mcp9600/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+CONFIG_SENSOR=y
+CONFIG_MCP9600=y
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_NEWLIB_LIBC=y
```

Comment:
You shouldn't need newlib now the Zephyr natively knows how to format floating point.  You do need  CBPRINTF_FP_SUPPORT.

Reply:
I will test it without that.

Reply:
done, tested